tests/manage/pv_services/test_run_io_multiple_pods.py

```diff
@@ -1,103 +1,52 @@
 import pytest
-import logging
-import random
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
 from ocs_ci.ocs import constants
-
 from ocs_ci.framework.testlib import ManageTest, tier2
-from tests.fixtures import (
-    create_rbd_storageclass, create_ceph_block_pool,
-    create_rbd_secret, create_pods, create_pvcs, create_project,
-    create_cephfs_secret, create_cephfs_storageclass
-)
-
-
-logger = logging.getLogger(__name__)
-
-
-@pytest.fixture()
-def init_pvc_size(request):
-    """
-    Initialize the PVC size for PVC creation
-    """
-    class_instance = request.node.cls
-    class_instance.pvc_size_int = getattr(
-        class_instance, 'pvc_size_int', random.randint(1, 10)
-    )
-    class_instance.pvc_size = f'{class_instance.pvc_size_int}Gi'
 
 
 @tier2
-@pytest.mark.usefixtures(
-    create_project.__name__,
-    init_pvc_size.__name__,
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-692")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-693")
+        )
+    ]
 )
-class BaseRunIOMultiplePods(ManageTest):
+class TestIOMultiplePods(ManageTest):
     """
     Run IO on multiple pods in parallel
     """
     num_of_pvcs = 10
-    pvc_size_int = 5
-    interface = None
+    pvc_size = 5
 
-    def run_io_multiple_pods(self):
-        """
-        Run IO on multiple pods in parallel
+    @pytest.fixture()
+    def pods(self, interface, pod_factory, multi_pvc_factory):
         """
+        Prepare multiple pods for the test
 
-        for pod in self.pod_objs:
-            pod.run_io('fs', f'{self.pvc_size_int - 1}G')
+        Returns:
+            list: Pod instances
 
```

Comment:
Blank line can be removed.

Reply:
No need, see https://www.sphinx-doc.org/en/1.5/ext/example_google.html